content/spin/v2/rust-components.md

```diff
@@ -342,8 +336,8 @@ fn publish(_req: Request) -> Result<Response> {
 
     // Publish to Redis
     match redis::publish(&address, &channel, &payload) {
-        Ok(()) => Ok(http::Response::builder().status(200).body(None)?),
-        Err(_e) => internal_server_error(),
+        Ok(()) => Ok(Response::builder().status(200).build()),
```

Comment:
Did we have a PR in flight for implementing IntoResponse on the builder type?

Reply:
Unfortunately `.status` returns `&mut ResponseBuilder` and not `ResponseBuilder`